config/routes.rb

```diff
@@ -1064,7 +1064,7 @@ def patch(*)
 
     resources :associated_groups, only: %i[index], constraints: AdminConstraint.new
 
-    get "slugs/generate", to: "slugs#generate"
+    post "slugs", to: "slugs#generate"
```

Comment:
Sorry nitpicking here, but convention should be #create

Reply:
I get that, but feels really weird to call this `create` since we are not actually creating anything...this is also why I originally had it as GET. I am going to leave it as is as a kind of middle ground :) I guess we can revisit if there are more actions added to this controller, but I don't think it's that bad atm.